source/extensions/filters/http/ext_proc/ext_proc.h

```diff
@@ -305,7 +306,17 @@ class FilterConfig {
 
   std::unique_ptr<OnProcessingResponse> createOnProcessingResponse() const;
 
+  Http::Code statusOnError() const { return status_on_error_; }
+
 private:
+  static Http::Code toErrorCode(uint64_t status) {
+    const auto code = static_cast<Http::Code>(status);
+    if (code >= Http::Code::Continue && code <= Http::Code::NetworkAuthenticationRequired) {
```

Comment:
I can't think of a legitimate case where someone would return a 1xx here so I would probably just make it generic and probably only allow 4xx or 5xx. But up to you.

Reply:
Thanks for the review @mattklein123. Making it only 4XX/5XX make sense to me. I changed it.